R/utils.R

```diff
@@ -781,3 +781,57 @@ calc_sd <- function(x) {
 get_env_var <- function(x, ...) {
   gsub("\\\\", "", Sys.getenv(x, ...))
 }
+
+#' Remove batch substring from drug id
+#'
+#' Gnumber, i.e. "G[0-9]{8}" is currently the default format of drug_id. It's also used as a drug name in some cases.
```

Comment:
Maybe we could use a example instead of regex expression?
```suggestion
#' Gnumber, i.e. "G12345678" is currently the default format of drug_id. It's also used as a drug name in some cases.
```